src/Microsoft.VisualStudio.Services.Agent/JobNotification.cs

```diff
@@ -12,23 +12,28 @@ namespace Microsoft.VisualStudio.Services.Agent
     [ServiceLocator(Default = typeof(JobNotification))]
     public interface IJobNotification : IAgentService, IDisposable
     {
-        Task JobStarted(Guid jobId);
+        Task JobStarted(Guid jobId, string accessToken, Uri serverUrl);
         Task JobCompleted(Guid jobId);
-        void StartClient(string pipeName, CancellationToken cancellationToken);
-        void StartClient(string socketAddress);
+        void StartClient(string pipeName, string monitorPort, CancellationToken cancellationToken);
```

Comment:
are we still using pipeName?

Reply:
I'm not sure of all the MMS flows, just wanted to make sure the monitor works irrespective of the communication protocol.